common/lib/types/message.js

```diff
@@ -18,12 +18,12 @@ var Message = (function() {
 	Message.prototype.toJSON = function() {
 		var result = {
 			name: this.name,
-			clientId: this.clientId,
-			connectionId: this.connectionId,
```

Comment:
It is valid to include a `connectionId` in `Message` when sent over REST so I don't think it's valid to remove it here.